pages/common/dfc.md

```diff
@@ -0,0 +1,20 @@
+# dfc
+
+> Gives an overview of the file system disk space usage with colours and graphs.
```

Comment:
Fixed.

This would probably also need to be fixed over in the [df page](https://github.com/tldr-pages/tldr/blob/master/pages/common/df.md). I could open a second PR to fix it there as well while I'm at it, should I?

Reply:
Go for it!

Reply:
Ok so I probably should have looked into this more before going straight to committing it. I did some looking around and from what I can tell both "file system" and "filesystem" are commonly used. I didn't find many dictionary entries for either.

Firefox tells me "filesystem" is wrong and should be "file system". The archwiki page for [file system](https://wiki.archlinux.org/index.php/File_systems) consistently uses "file system" but if you go to the [ext4](https://wiki.archlinux.org/index.php/Ext4) page there they use "filesystem".

I think the best thing to do would be use whatever is currently used more in the existing tldr pages so I very quickly counted all lines with occurrences.
![zsh shell showing 141 occurrences for "filesystem" and 17 for "file system"](https://user-images.githubusercontent.com/10325316/94919496-dccd5280-04b4-11eb-9c8f-0e6ddc2bde2b.png)

141 occurrences of "filesystem" and 17 for "file system" so based on that doing what you said makes the most sense. I thought I'd add this comment for extra clarity but there shouldn't be a need for another commit (regarding this specifically).

Reply:
@sbrl ?

Reply:
Oh, that is odd. Yeah, we standardised on `filesystem` a little while ago (though I forget the PR), so let's use that.

It's interesting to see that different places add a space, and others don't.